gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/task/AcknowledgmentMessageHandler.java

```diff
@@ -82,7 +89,6 @@ private MigrationStatus getMigrationStatus(String ackTypeCode, String reasonCode
                 case "18" -> EHR_EXTRACT_REQUEST_NEGATIVE_ACK_GP2GP_MISFORMED_REQUEST;
                 case "19" -> EHR_EXTRACT_REQUEST_NEGATIVE_ACK_GP2GP_NOT_PRIMARY_HEALTHCARE_PROVIDER;
                 case "24" -> EHR_EXTRACT_REQUEST_NEGATIVE_ACK_GP2GP_MULTI_OR_NO_RESPONSES;
-                case "99" -> EHR_EXTRACT_REQUEST_NEGATIVE_ACK_UNKNOWN;
```

Comment:
While I can see this is unnecessary, I quite like that it provides documentation that 99 is a valid value.

Reply:
I agree with this, it think it's useful to retain this